comparisons/CompareGraphs.py

```diff
@@ -83,7 +85,8 @@
         hToCompare[iFile].SetStats(0)
     else:
         hToCompare[iFile].SetName(f'g{iFile}')
-    SetObjectStyle(hToCompare[iFile], color=GetROOTColor(color), markerstyle=GetROOTMarker(marker), fillstyle=0)
+    SetObjectStyle(hToCompare[iFile], color=GetROOTColor(color), markerstyle=GetROOTMarker(marker), markersize=inputCfg['options']['markersize'],
```

Comment:
Minor comment: likely it is longer than 120 columns, maybe you can add a newline after `markerstyle=GetROOTMarker(marker),`

Reply:
Sure! (Also for this, once for all the similar comment)